src/cmd/nsfs.js

```diff
@@ -206,6 +206,7 @@ class NsfsObjectSDK extends ObjectSDK {
             },
             system_owner: new SensitiveString('nsfs'),
             bucket_owner: new SensitiveString('nsfs'),
+            owner_account: new SensitiveString('nsfs-id'), // temp
```

Comment:
`owner_account` is failing to convey the actual value stored in this property. Can We have something like `bucket_owner_id`. WDYT?

Reply:
The change of `owner_account` was added after the change of the function `read_bucket_sdk_policy_info` in which I added the `owner_account` as a property of the returned object. The property `owner_account` is part of the NSFS bucket schema:
https://github.com/noobaa/noobaa-core/blob/426e7c79f33245c765a39a9ee0aaebc84e16f7cc/src/server/system_services/schemas/nsfs_bucket_schema.js#L22-L25

If you want to change the property name from `owner_account ` to `bucket_owner_id` it is something that we can do, but it is a schema change and should be in a separate PR, not related to this fix.